crates/forge/bin/cmd/install.rs

```diff
@@ -19,6 +20,8 @@ use yansi::Paint;
 static DEPENDENCY_VERSION_TAG_REGEX: LazyLock<Regex> =
     LazyLock::new(|| Regex::new(r"^v?\d+(\.\d+)*$").unwrap());
 
+pub const FORGE_SUBMODULES_INFO: &str = "forge-submodules-info.json";
```

Comment:
Can we rename this `foundry.lock`? Considering it effectively serves as a lockfile not intended to be manually edited and checked in

Reply:
to add to this point, should we maybe put it in `lib` dir instead prj root?

Reply:
I think it makes sense to put it in the root as it is related to `.gitmodules`